### ISSUE TITLE
Feature: Character Variant for `l` and `1` in Neon and Argon

### ISSUE THREAD

**Issue by pkj-l** [githubnext/monaspace#62]
Thanks for your amazing typefaces.
`1` and `l` looks silimar in Neon and Argon.
![1 and l in Neon](https://github.com/githubnext/monaspace/assets/43498546/893677ed-7b37-431f-8e05-acfc4410925b)
![1 and l in Argon](https://github.com/githubnext/monaspace/assets/43498546/1cf35410-f456-4080-9cfa-c6452d8608ab)
I’d like an alternative character for `l` with a tail (like JetBrains Mono) or an alternative character for `1` without bottom slab (like Roboto Mono and Inconsolata) .
![JetBrains Mono](https://github.com/githubnext/monaspace/assets/43498546/fa9ee766-ef4a-4e99-b9d4-5c8cb3c3d592)
![Roboto Mono and Inconsolata](https://github.com/githubnext/monaspace/assets/43498546/9907c828-6ed7-4d05-8c8d-7a941838f72c)
Thanks!

**Comment by LeoVasanko** [comment]
Yes, the confusing characters should be made look different. On Radon `l` looks quite a bit like `2` even. Hoping to see all these little issues be ironed out. Thanks for the great fonts! :)

**Comment by chtenb** [comment]
![image](https://github.com/githubnext/monaspace/assets/1030961/242a1085-9368-405b-9d5c-25bbb6612bc2)

This looks like it says `12`, but it's actually `l2`

**Comment by idan** [comment]
Added in 1.2!
